### PR TITLE
Add Russian language support

### PR DESCRIPTION
**Pull request by SokolovskyMaxim**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+msgid "New _Section"
+msgstr "Новый _Раздел"
+
+#: src/ui/window.ui:183
+msgid "_Edit Project"
+msgstr "_Редактировать Проект"
+
+#: src/ui/window.ui:189 src/ui/window.ui:211
+msgid "_Preferences"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:319]: ☝🏻

File: po/ru.po [hunk 2]
```diff
@@ -0,0 +1,707 @@
...
+msgid "_Keyboard Shortcuts"
+msgstr "_Комбинации клавиш"
+
+#: src/ui/window.ui:201 src/ui/window.ui:223
+msgid "_About IPlan"
+msgstr "_О приложении"
+
+#: src/ui/preferences_window.ui:17
+msgid "Run in Background"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:335]: ☝🏻

File: po/ru.po [hunk 3]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/ui/preferences_window.ui:18
+msgid "Allow send reminders when the app is closed"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:339]: ☝🏻

File: po/ru.po [hunk 4]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/ui/shortcuts_window.ui:13
+msgctxt "shortcut window"
+msgid "Application"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:344]: ☝🏻

File: po/ru.po [hunk 5]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/ui/shortcuts_window.ui:16
+msgctxt "shortcut window"
+msgid "Preferences"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:349]: ☝🏻

File: po/ru.po [hunk 6]
```diff
@@ -0,0 +1,707 @@
...
+msgid "Backup"
+msgstr "Резервная копия"
+
+#: src/ui/backup_window.ui:35 src/views/backup_window.rs:62
+msgid "Export"
+msgstr "Экспорт"
+
+#: src/ui/backup_window.ui:47
+msgid "Data folder path"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:386]: ☝🏻

File: po/ru.po [hunk 7]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/ui/backup_window.ui:48
+msgid "Add this path to your backup application"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:390]: ☝🏻

File: po/ru.po [hunk 8]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/ui/backup_window.ui:68 src/views/backup_window.rs:98
+msgid "Import"
+msgstr "Импорт"
+
+#: src/ui/backup_window.ui:69
+msgid "While importing, previous data will be cached"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:398]: ☝🏻

File: po/ru.po [hunk 9]
```diff
@@ -0,0 +1,707 @@
...
+msgid "Archive"
+msgstr "Архивировать"
+
+#: src/ui/project/project_edit_window.ui:115
+msgid "Delete"
+msgstr "Удалить"
+
+#: src/ui/project/project_header.ui:108
+msgid "Time spent"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:450]: ☝🏻

File: po/ru.po [hunk 10]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/ui/project/project_header.ui:141
+msgid "Time spent per day"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:454]: ☝🏻

File: po/ru.po [hunk 11]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/ui/project/project_page.ui:34
+msgid "no sections found"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:458]: ☝🏻

File: po/ru.po [hunk 12]
```diff
@@ -0,0 +1,707 @@
...
+msgstr "Добавить задачу"
+
+#: src/ui/task/tasks_done_window.ui:50
+msgid "no done tasks found"
+msgstr "Нет выполненных задач"
+
+#. Translators: Replace "translator-credits" with your names, one name per line
+#: src/application.rs:314
+msgid "translator-credits"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:585]: Here you can add your name to the About page. in one of these formats:
```
Your Name https://example.com
Your Name <example@mail.com>
```

File: po/ru.po [hunk 13]
```diff
@@ -0,0 +1,707 @@
...
+msgid "until"
+msgstr "вместо"
+
+#: src/views/record/record_window.rs:120
+msgid "Edit Record"
+msgstr "Редактировать запись"
+
+#: src/views/record/record_window.rs:207
+msgid "Duration can't be 0"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:671]: ☝🏻

File: po/ru.po [hunk 14]
```diff
@@ -0,0 +1,707 @@
...
+
+#: src/views/record/record_window.rs:215
+msgid "Record end time can't be in future"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:675]: ☝🏻

File: po/ru.po [hunk 15]
```diff
@@ -0,0 +1,707 @@
...
+msgid "Task"
+msgstr "Задача"
+
+#: src/views/sidebar/sidebar_projects.rs:360
+msgid "Task moved to {}"
+msgstr "Задача перемещена в {}"
+
+#: src/views/sidebar/sidebar_projects.rs:374
+msgid "{} doesn't have any section"
+msgstr ""
```
Comment (iman-salmani) [inline po/ru.po:695]: ☝🏻

File: po/ru.po [hunk 16]
```diff
@@ -0,0 +1,707 @@
...
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: iplan\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-07-07 21:11+0330\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
```
Comment (iman-salmani) [inline po/ru.po:13]: You can also put your name here ☝🏻

**Review by iman-salmani — CHANGES_REQUESTED** [review]
Thank you for your contribution
meta info texts are unnecessary but some texts in UI seem to be forgotten. I pointed them out in the following messages.
Also, you can add your name below `translator-credits` message-id to be shown in the About window.

**Comment by SokolovskyMaxim** [comment]
> meta info texts are unnecessary

how should I distinguish meta information from the interface?

**Comment by iman-salmani** [comment]
metainfo.xml file contains app data to show in app stores and I'm not sure if I'm doing this wrong or if App stores like Flathub do not support translation for app info.
If they support them I will exclude the release notes part from the translation files.
And if not meta info will be excluded.

You can find out where text is used by looking top of the msgid in the po file.
In case you using Poedit, right-click on the text.

**Pull request merged**